dashboard-frontend/src/pages/StudyConfPage/configs/general.js

```diff
@@ -0,0 +1,107 @@
+export const general = {
+  type: 'confObject',
+  title: 'General',
+  description: 'The "general" configuration consists of... General stuff?',
+  fields: [
```

Comment:
Do we know what the actual values we could use for these are?

i.e maybe we can check our current data to find out what possible values we could have for:
- `optimization_goals`
- `objective`